src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -161,21 +191,81 @@ class RuleNode extends React.Component<Props> {
     );
   }
 
+  conditionallyRenderHelpfulBanner() {
+    const {data, project, organization} = this.props;
+    /**
+     * Would prefer to check if data is of `IssueAlertRuleAction` type, however we can't do typechecking at runtime as
+     * user defined types are erased through transpilation.
+     * Instead, we apply duck typing semantics here.
+     * See: https://stackoverflow.com/questions/51528780/typescript-check-typeof-against-custom-type
+     */
+    if (!data?.targetType) {
+      return null;
+    }
+
+    switch (data.targetType) {
+      case MailActionTargetType.IssueOwners:
+        return (
+          // Using boolean attributes on styled components cases a warning
```

Comment:
I think this means that Alert is passing the attribute to the DOM node and throws a warning. In these cases we generally make sure that these props do not get passed, or if the styled component needs it, we use emotion's `shouldForwardProp` option and `isValidProperty` helper (not sure if these are the right names, but they're close)

Reply:
Opted to go with the current code style and prevented prop forwarding by adding `thinner` to the list of parameters